### PR TITLE
Update glossary entry for Quantity

### PR DESCRIPTION
**Pull request by KhushPatibandha**
Issue - #30885 
This is my first PR, do let me know if there are any changes required.

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Update glossary entry for Quantity

Still relevant to #30885

File: content/en/docs/reference/glossary/quantity.md [hunk 1]
```diff
@@ -30,3 +30,4 @@ intentionally lowercase), `M` (mega), `G` (giga), `T` (tera), `P` (peta),
 The accepted binary (power-of-2) units are `Ki` (kibi), `Mi` (mebi), `Gi` (gibi),
 `Ti` (tebi), `Pi` (pebi), `Ei` (exbi).
 
+Some terminologies - `mB` stands for millibyte, `KB` stands for kilobyte, `MB` stands for megabyte, `GB` stands for gigabyte, `TB` stands for terabyte, `PB` stands for petabyte, `EB` stands for exabyte, `KiB` stands for kibibyte, `MiB` stands for mebibyte, `GiB` stands for gibibyte, `TiB` stands for tebibyte, `PiB` stands for pebibyte, `EiB` stands for exbibyte
```
Comment (NitishKumar06) [inline content/en/docs/reference/glossary/quantity.md:33]: Could we add all these under bullet points? WDYT?

**Review by NitishKumar06 — COMMENTED** [review]
The checks are failing. Try pushing a blank commit!

File: content/en/docs/reference/glossary/quantity.md (hunk 1 above)
Comment (KhushPatibandha) [reply content/en/docs/reference/glossary/quantity.md:33]: Hey, @NitishKumar06 I can surely add all these as bullet points

**Pull request closed without merging**